help/en/releasenotes/current-draft-note.shtml

```diff
@@ -424,7 +424,10 @@
    <h3>Dispatcher</h3>
         <a id="Dispatcher" name="Dispatcher"></a>
         <ul>
-             <li></li>
+             <li>Add minimum reliable operating throttle setting.
+             calculate adjusted throttle as throttle * factor then apply max and min limits.
```

Comment:
No need to change, but the first letter in the sentence should be upper case. Se the word "calculate".